installer/conf/logrotate.conf

```diff
@@ -6,3 +6,12 @@
     size 100M
     copytruncate
 }
+
+/var/opt/microsoft/omsagent/log/update_run_progress.log {
+    rotate 1
```

Comment:
Are you sure that this will be enough information to maintain after the logs are rotated? I would suggest at least 3 zipped old logs, just in case of a disaster.